ingestion/src/metadata/ingestion/source/database/azuresql/metadata.py

```diff
@@ -20,7 +22,43 @@
     Source as WorkflowSource,
 )
 from metadata.ingestion.api.source import InvalidSourceException
+from metadata.ingestion.source.database.column_type_parser import create_sqlalchemy_type
 from metadata.ingestion.source.database.common_db_source import CommonDbSourceService
+from metadata.ingestion.source.database.mssql.utils import (
+    get_columns,
+    get_table_comment,
+    get_view_definition,
+)
+from metadata.utils.sqlalchemy_utils import (
+    get_all_table_comments,
+    get_all_view_definitions,
+)
+
+ischema_names.update(
+    {
+        "nvarchar": create_sqlalchemy_type("NVARCHAR"),
```

Comment:
@ulixius9 can we make sure in our tests we create tables with all of these types for each source?

Reply:
many types are already covered, will open a separate pr to cover the remaining types!